client/components/editor/index.js

```diff
@@ -67,16 +67,12 @@ class TextEditor extends Component {
   };
 
   onBlur = (self, editor, next) => {
-    next();
+    next()
```

Comment:
```suggestion
    next();
```

Reply:
there are about a billion of these in projects, as our linting doesn't care, we should make it care, but let's do them all together?